apps/xmtp.chat/src/stores/profiles.ts

```diff
@@ -59,7 +59,7 @@ export const profilesStore = createStore<ProfilesState & ProfilesActions>()(
       const existingProfiles = state.profiles.get(profile.address) ?? [];
       newProfiles.set(profile.address, [...existingProfiles, profile]);
       if (profile.identity) {
-        newNames.set(profile.identity, profile.address);
+        newNames.set(profile.identity.toLowerCase(), profile.address);
```

Comment:
we can also handle this case in the API service, but i'm not sure that would be necessary if ENS/base names are always lowercase.

Reply:
I want to fix this UX issue:

<img width="1325" height="677" alt="image" src="https://github.com/user-attachments/assets/6d06effe-3758-4edf-a032-b6fce55c778d" />